src/components/stepforms/AggregateStepForm.vue

```diff
@@ -2,7 +2,7 @@
   <div>
     <StepFormHeader :title="title" :stepName="this.editedStep.name" />
     <MultiselectWidget
-      id="groupbyColumnsInput"
```

Comment:
Hum, In AutocompleteWidget, `id` is used as a props, i think it will break something to rename it `class`?

Reply:
Yes you are right: here is the line where I delete it: https://github.com/ToucanToco/weaverbird/pull/489/files#diff-0897a479bdd646438d8b6234e5d0b092L33

Reply:
Ok, it stay the `:for="id"` here https://github.com/ToucanToco/weaverbird/pull/489/files#diff-0897a479bdd646438d8b6234e5d0b092R3
But I don't know what it's used for.

Reply:
It's used to specifies which form element a label is bound to. I will find a way to overcome this.

Reply:
Apparently the label is bind to nothing so I can remove it :)